Mathlib/RingTheory/TensorProduct/Basic.lean

```diff
@@ -923,6 +931,16 @@ theorem map_range (f : A →ₐ[R] B) (g : C →ₐ[R] D) :
   · rw [← map_comp_includeLeft f g, ← map_comp_includeRight f g]
     exact sup_le (AlgHom.range_comp_le_range _ _) (AlgHom.range_comp_le_range _ _)
 
+lemma comm_comp_map (f : A →ₐ[R] C) (g : B →ₐ[R] D) :
+    (TensorProduct.comm R C D).toAlgHom.comp (Algebra.TensorProduct.map f g) =
```

Comment:
Do you want `AlgEquiv.toAlgHom` or `AlgHomClass.toAlgHom` here?

Reply:
Good point, I will change it to `(TensorProduct.comm R C D : C ⊗[R] D →ₐ[R] D ⊗[R] C)`